kdwsdl2cpp/src/settings.cpp

```diff
@@ -205,3 +205,18 @@ void Settings::setNameSpace(const QString &ns)
 {
     mNameSpace = ns;
 }
+
+QStringList Settings::importPathList() const
+{
+    return mImportPathList;
+}
+
+void Settings::setImportPathList(const QStringList &importPathList)
+{
+    mImportPathList = importPathList;
+    if (mWsdlUrl.isLocalFile()) {
+        QFileInfo wsdlFileInfo(mWsdlUrl.path());
+        QString wsdlDirPath = wsdlFileInfo.dir().path();
```

Comment:
wsdlFileInfo.absolutePath() would be faster, no need for the intermediate QDir object.